src/Logging/NullScope.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Tools.Logging
+{
+    internal class NullScope : IDisposable
```

Comment:
what is this used for?

Reply:
Used by the Loggers. It used to be a public type but has disappeared from the new version of the library.